tests/ludwig/utils/test_backward_compatibility.py

```diff
@@ -542,43 +540,6 @@ def test_update_increase_batch_size_on_plateau_max():
     assert updated_config == expected_config
 
 
-def test_old_class_weights_default():
-    old_config = {
-        "input_features": [
-            {
-                "name": "input_feature_1",
-                "type": "category",
-            }
-        ],
-        "output_features": [
-            {"name": "output_feature_1", "type": "category", "loss": {"class_weights": 1}},
-        ],
-    }
-
-    new_config = {
-        "input_features": [
-            {
-                "name": "input_feature_1",
-                "type": "category",
-            }
-        ],
-        "output_features": [
-            {"name": "output_feature_1", "type": "category", "loss": {"class_weights": None}},
-        ],
-    }
-
-    upgraded_config = upgrade_config_dict_to_latest_version(old_config)
```

Comment:
I think we'd still wanna test the upgrade, no?

Reply:
Ah good looks, I removed this because I removed the backwards compatibility step before, but added it back in and changed it per another issue. Will add this back in today!